_disability-benefits/claims-appeal.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Appeals
-concurrence: incomplete
+concurrence: complete
 template: 6-info-page
 ---
```

Comment:
I think this pull request was likely intended to be off of master, not the careeers and employment branch. I'm going to remake it, close this one, and assign to @bethpotts